solutions/kubernetes/cluster-functions.sh

```diff
@@ -219,7 +219,7 @@ function install_prerequisites() {
             CALICO_IMAGE_VERSION=$(grep 'docker.io/calico/cni' calico-$CALICO_PLUGIN_VERSION.yaml | uniq | awk -F':' '{ print $3}')	
             wget -c https://github.com/projectcalico/calico/releases/download/$CALICO_IMAGE_VERSION/release-$CALICO_IMAGE_VERSION.tgz -O - | tar -xz || throw $Exception
             pushd release-$CALICO_IMAGE_VERSION/images 
-                ls *tar | xargs -I{} docker image load -i {} || throw $Exception
+                ls ./*tar | xargs -I{} docker image load -i {} || throw $Exception
```

Comment:
Please check that the scripts work with this change

Reply:
Removed this file